pkg/ipam/poolallocator/allocator.go

```diff
@@ -540,11 +567,9 @@ func (a *IPPoolAllocator) ReleaseContainer(containerID, ifName string) error {
 	return err
 }
 
-// HasResource checks whether an IP was associated with specified pod. It returns error if the resource is crd fails to be retrieved.
+// HasResource checks whether an IP was associated with specified pod. It returns error if the IPPool CR fails to be retrieved.
```

Comment:
Not introduced by this PR, but could be corrected with the change. s/HasResource/HasPod/

Reply:
Sure. Let me change it.